<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMessageDialog</class>
 <widget class="QDialog" name="SendMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отправить сообщение</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/send_message.png</normaloff>:/icons/icons/send_message.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="rowWrapPolicy">
    <enum>QFormLayout::DontWrapRows</enum>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="subject_label">
     <property name="text">
      <string>Тема сообщения:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="subject_line_edit">
     <property name="text">
      <string>Привет, мир! Hello, world!</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="recipients_label">
     <property name="text">
      <string>Получатели:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="recipients_line_edit">
     <property name="text">
      <string>robert.wayne.alx@yandex.ru, robert.wayne.alx@ya.ru</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="cc_recipients_label">
     <property name="text">
      <string>Получатели копии:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="cc_recipients_line_edit">
     <property name="text">
      <string>robert.wayne.alx@yandex.by, robert.wayne.alx@yandex.com</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="bcc_recipients_label">
     <property name="text">
      <string>Получатели скрытой копии:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="bcc_recipients_line_edit">
     <property name="text">
      <string>robert.wayne.alx@yandex.ua, robert.wayne.alx@yandex.kz</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="content_label">
     <property name="text">
      <string>Текст сообщения:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="content_text_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="plainText">
      <string>Привет, мир!
Hello, world!
</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="attachment_label">
     <property name="text">
      <string>Вложения:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="add_attachment">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Добавить вложение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_attachment">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Удалить вложение</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QTableWidget" name="attachments_table">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Имя файла</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Размер файла</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Путь к файлу</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="note_label">
     <property name="text">
      <string>(получатели указываются через запятую либо пробел)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>SendMessageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>SendMessageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
